react/src/components/molecules/Tooltip/style.scss

```diff
@@ -0,0 +1,172 @@
+@import 'global';
+
+.ma__tooltip {
```

Comment:
are these styles direct copy from assets/scss? if so, can you use @import instead